.changeset/wicked-cups-search.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/firestore': minor
+---
+
+Removed validation of `null` and `NaN` values in filters. However, note that only `==` and `!=` filters match against `null` and `NaN`.
```

Comment:
> However, note that only `==` and `!=` filters match against `null` and `NaN`.

This reads to me in a way that suggests we're removing guard rails but still expecting users to remember this. I'd suggest something more like this:

> Removed excess validation of `null` and `NaN` values in query filters. This more closely aligns the SDK with the Firestore backend, which has always accepted `null` and `NaN` for all operators, even though this isn't necessarily useful.